NitroxServer/Server.cs

```diff
@@ -44,11 +47,53 @@ public Server(ServerConfigReader configReader)
 
         public void Start()
         {
+            ListServerIPs();
```

Comment:
I think we should put this into a new class: IpResolver (return List<String>) or IpLogger (print internally).